pkg/storage/poststorage/redis_delete.go

```diff
@@ -0,0 +1,19 @@
+package poststorage
+
+import (
+	"github.com/uvio-network/apiserver/pkg/format/storageformat"
+	"github.com/xh3b4sd/tracer"
+)
+
+func (r *Redis) DeleteExpiry(inp []*Object) error {
+	var err error
+
+	for i := range inp {
+		err = r.red.Sorted().Delete().Value(storageformat.PostExpiry, inp[i].ID.String())
```

Comment:
Removing an ID from the index is basically the reverse operation of adding it like we do in `redis_create.go`. 

```go
err = r.red.Sorted().Create().Score(storageformat.PostExpiry, inp[i].ID.String(), float64(inp[i].Expiry.UnixNano()))
```

Elements in a sorted set are key value pairs, where the key is a string and the value is a number, the score. We can decide here whether we want to cleanup using the key or the value. We could also do `Sorted().Delete().Score` and use the expiry as second argument instead of the ID string. My thinking here for deleting by key instead of value was that the ID is supposed to be more unique than something that happens at the same time. We could imagine that multiple claims expire on the exact same date. If we were to delete elements from the sorted set by score, then we would remove all elements matching that score. Our objective here is to only delete the claims that we have been given, and so we delete their indizes using their string value in that sorted set. This allows us to only delete the elements that we are in fact targeting.